### DISCUSSION TITLE
When I am trying to deploy a derivative function I get error:

### DISCUSSION THREAD

**Discussion by khomya** [AntlerVC/firetable#1422] [Support / Q&A]
=== Deploying to 'georgia-tennis'...

i  deploying functions
i  functions: preparing codebase default for deployment
i  functions: ensuring required API cloudfunctions.googleapis.com is enabled...
i  functions: ensuring required API cloudbuild.googleapis.com is enabled...
i  artifactregistry: ensuring required API artifactregistry.googleapis.com is enabled...
✔  functions: required API cloudfunctions.googleapis.com is enabled
✔  functions: required API cloudbuild.googleapis.com is enabled
✔  artifactregistry: required API artifactregistry.googleapis.com is enabled

Error: Failed to load function definition from source: Failed to generate manifest from function source: SyntaxError: Cannot use import statement outside a module
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
38
12:03:59:59

command: cd build/functionBuilder/builds/1694592182536; yarn deploy --project georgia-tennis --only functions

stderr: error Command failed with exit code 1.

errorStackTrace: Error: Command failed: cd build/functionBuilder/builds/1694592182536; yarn deploy --project georgia-tennis --only functions
error Command failed with exit code 1.

    at ChildProcess.exithandler (node:child_process:419:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
    
    
  --------------------------------------------  
    I try to do the simplest example ever:

const derivative: Derivative = async ({ row, ref, db, storage, auth, logging }) => {
return "123";

}

listening on toggle field. Just like in video lesson: https://docs.rowy.io/field-types/derivative

**Comment by khomya** [comment]
my account: khomya@gmail.com

**Reply by Chadha93** [reply]
Thank you for providing the email, we'll look into this issue and will update here soon. cc @BeeBombshell.

I invite you to join our [Discord](https://www.rowy.io/discord) and post this on the community support channel. 

--
Cheers,
Gaurav Chadha | Technical Support Engineer | [Rowy](https://rowy.app/) + [BuildShip](https://buildship.com/)
backend for everyone
